cfpb/cfgov-refresh
# Adding link button option to text introduction

Adding link button option to text introduction

## Changes

- Modified `cfgov/jinja2/v1/_includes/molecules/text-introduction.html` to add `link_is_button` boolean option. 

## Testing

- Create a sub-landing page.
- Add a text introduction to the page.
- Add a link.
- Check the `link_is_button` flag.
- Verify that the button is operable and looks as expected.

## Screenshots

<img width="811" alt="screen shot 2017-04-14 at 12 14 05 pm" src="https://cloud.githubusercontent.com/assets/1696212/25048746/01919c9c-210c-11e7-878a-c2b732c335ff.png">

## Notes

- This is being suggested by the Mosaic team. This code is essentially the same as within the info unit.

## Todos

-

## Checklist
